sophos_central/help.md

```diff
@@ -41,6 +42,41 @@ Example input:
 
 ### Actions
 
+#### Check Tamper Protection Status
+
+This action is used to get tamper status by IP address, Hostname, MAC address or Device ID.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|agent|string|None|True|Device ID, IPv4 address, IPv6 address, MAC address or hostname|None|198.51.100.100|
+
+Example input:
+
+```
+{
+  "agent": "198.51.100.100"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|tamper_status|tamper_protection_entity|True|Tamper status for provided agent|
+
+Example output:
+
+```
+{
+  "tamper_status": {
+    "enabled": true,
+    "password": "541298753620",
```

Comment:
@r7-kszczepanskagorna can we update this plugin to only return `enabled`?

We don't want to capture the passwords and previousPasswords in-product for security leaks.